packages/stardog-language-utils/src/common-completions.ts

```diff
@@ -0,0 +1,229 @@
+    'FunctionalProperty',
+    'InverseFunctionalProperty',
+    'IrreflexiveProperty',
+    'NamedIndividual',
+    'NegativePropertyAssertion',
+    'Nothing',
+    'ObjectProperty',
+    'Ontology',
+    'OntologyProperty',
+    'ReflexiveProperty',
+    'Restriction',
+    'SymmetricProperty',
+    'TransitiveProperty',
+    'Thing',
+  ],
+  properties: [
+    'allValuesFrom',
+    'annotatedProperty',
+    'annotatedSource',
+    'annotatedTarget',
+    'assertionProperty',
+    'backwardCompatibleWith',
+    'bottomDataProperty',
+    'bottomObjectProperty',
+    'cardinality',
+    'complementOf',
+    'datatypeComplementOf',
+    'deprecated',
+    'differentFrom',
+    'disjointUnionOf',
+    'disjointWith',
+    'distinctMembers',
+    'equivalentClass',
+    'equivalentProperty',
+    'hasKey',
+    'hasSelf',
+    'hasValue',
+    'imports',
+    'incompatibleWith',
+    'intersectionOf',
+    'inverseOf',
+    'maxCardinality',
+    'maxQualifiedCardinality',
+    'members',
+    'minCardinality',
+    'minQualifiedCardinality',
+    'onClass',
+    'onDataRange',
+    'onDatatype',
+    'oneOf',
+    'onProperties',
+    'onProperty',
+    'priorVersion',
+    'propertyChainAxiom',
+    'propertyDisjointWith',
+    'qualifiedCardinality',
+    'sameAs',
+    'someValuesFrom',
+    'sourceIndividual',
+    'targetIndividual',
+    'targetValue',
+    'topDataProperty',
+    'topObjectProperty',
+    'unionOf',
+    'versionInfo',
+    'versionIRI',
+    'withRestrictions',
+  ],
+};
+
+const xsd: CompletionDatum = {
+  namespaceIri: 'http://www.w3.org/2001/XMLSchema#',
+  datatypes: [
+    'ENTITIES',
+    'ENTITY',
+    'ID',
+    'IDREF',
+    'IDREFS',
+    'NCName',
+    'NMTOKEN',
+    'NMTOKENS',
+    'NOTATION',
+    'Name',
+    'QName',
+    'anyAtomicType',
+    'anySimpleType',
+    'anyType',
+    'anyURI',
+    'base64Binary',
+    'boolean',
+    'byte',
+    'date',
+    'dateTime',
+    'dateTimeStamp',
+    'dayTimeDuration',
+    'decimal',
+    'double',
+    'duration',
+    'float',
+    'gDay',
+    'gMonth',
+    'gMonthDay',
+    'gYear',
+    'gYearMonth',
+    'hexBinary',
+    'int',
+    'integer',
+    'language',
+    'long',
+    'negativeInteger',
+    'nonNegativeInteger',
+    'nonPositiveInteger',
+    'normalizedString',
+    'positiveInteger',
+    'short',
+    'string',
+    'time',
+    'token',
+    'unsignedByte',
+    'unsignedInt',
+    'unsignedLong',
+    'unsignedShort',
+    'yearMonthDuration',
+  ],
+  classes: [],
+  properties: [],
+};
+
+const commonCompletions = {
+  owl,
+  rdf,
+  rdfs,
+  xsd,
+};
+
+export const commonCompletionItems = Object.keys(commonCompletions).reduce(
+  (acc, key) => {
+    const { namespaceIri, classes = [], properties = [] } = commonCompletions[
+      key
+    ] as CompletionDatum;
+    const classCompletions: CompletionItem[] = classes.map((className) =>
+      makeCompletionItemFromPrefixedNameAndNamespaceIri(
+        `${key}:${className}`,
```

Comment:
this assumes the user does not change the namespace prefixes, which might usually be the case, but i think at least worth a comment that it's possible these suggestions are inaccurate as they ignore the database's known prefixes.

Reply:
Good call. In the SHACL language server, we actually make the prefix injectable into the parser, so I could perhaps do something like that here, too. I'll look into it.

Reply:
Actually, since this is intended to work with these particular namespaces because they are the ones loaded into Stardog by default (and because adjusting this for the abnormal case may be a decent chunk of work), I'm inclined to leave this as-is and just add a comment for now. Let me know if you oppose that, though; I could maybe be persuaded. My suspicion is that the abnormal case happens almost never.

Reply:
A comment makes sense to me.

Reply:
Actually, a little more thought has me thinking that it's easy to do, so let me try that first...

Reply:
![image](https://user-images.githubusercontent.com/8409748/70181645-4ec7e600-16b0-11ea-922d-e2ac49f27e1c.png)